app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -292,7 +293,7 @@ logger "Starting anaconda <%= @host %> postinstall"
 <%= snippet 'epel' -%>
 <% end -%>
 
-<%= snippet 'redhat_register' %>
+<%= snippet 'redhat_register' if rhel_compatible && os_major < 9 -%>
```

Comment:
This also includes syspurpose, but the [syspurpose](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#syspurpose) statement isn't in the new snippet. Is that something that should no longer be supported on EL9+?